uv.lock

```diff
@@ -0,0 +1,779 @@
+version = 1
+requires-python = ">=3.8"
+
+[[package]]
+name = "bcrypt"
+version = "4.2.1"
```

Comment:
We should not check in the lock file.
There are pros and cons to lock files for libraries, though at any rate, it should not be part of this PR.